Description: remove remote png from README
 especially since it makes marked-man generate a bad man page
Last-Update: 2016-11-11
Author: Jérémy Lal <kapouer@melix.org>
Forwarded: https://github.com/kapouer/marked-man/issues/7
--- a/README.md
+++ b/README.md
@@ -1,5 +1,3 @@
-[![NPM](https://nodei.co/npm/marked-man.png?downloads=true)](https://nodei.co/npm/marked-man/)
-
 marked-man(1) -- markdown to roff
 =================================
 
